src/components/CurriculumComponents/UnitsTab/UnitsTab.test.tsx

```diff
@@ -467,6 +475,62 @@ describe("components/pages/CurriculumInfo/tabs/UnitsTab", () => {
     });
   });
 
+  test("user can filter units by discipline", async () => {
```

Comment:
I would rather we had smaller components so that the year section could be tested in isolation. But for the moment I've selected the year within this test to reduce the noise.